### ISSUE TITLE
Downsize 404 web server warns to a debug 

### ISSUE THREAD

**Issue by markelog** [karma-runner/karma#1611]
In angular, there is hack for log4js logger - https://github.com/angular/angular.js/blob/8226ff8b8ec262a614ddb7635c33abf8e5624c67/karma-shared.conf.js#L163-165, it was done to partially hide 404 responses from the server.

This is pretty common situation and annoying one too.

You recently improved the situation by serving favicon, but test generated 404's couldn't be hidden without hacks, or custom log4js appender, which would also be a hack, since that appender, would need to use standart `console` appender, which is a karma dependency, so you would need to actually add your own log4js dependency to do it right.

I'm wondering if there is way to do it properly. One solution would to show those 404 only at debug level. Other is one i can think of, is to provide additional option to the config which would allow to specify array of such exceptions.  

Is there any better way?

**Comment by maksimr** [comment]
@markelog you can set **logLevel** to equal **LOG_ERROR**:

```json
...
 logLevel: config.LOG_ERROR,
...
```

**Comment by markelog** [comment]
Then i would miss actual needed `warns` and `info` messages

**Comment by maksimr** [comment]
@markelog yup. But sometimes `404` is important information

**Comment by maksimr** [comment]
Maybe we really should put it under `DEBUG` level ...

\cc @Dignifiedquire

**Comment by markelog** [comment]
Yes, but most times it is annoying or worrisome information which turns to be nothing.

Hacks like in angular is pretty common, i already implemented in couple projects, which is pretty meh...
